code/modules/mob/living/carbon/human/species/vox.dm

```diff
@@ -182,3 +225,207 @@
 
 /datum/species/skellington/skelevox/fallback()
 	return "Vox"
+
+/mob/living/carbon/human
+	var/feather_regen_timer = null
+	var/original_vox_s_tone = null
+	var/vox_full_regen_active = null
```

Comment:
These should be defined in the main /mob/living/carbon/human definition instead of here.